### PR TITLE
docs: update getting started

### PR DESCRIPTION
**Pull request by hilakashai**
Edited the quick introduction - have not removed applying & inspecting schema docs yet (only removed from sidebar)

### CODE REVIEW

File: doc/md/getting-started/getting-started.mdx [hunk 1]
```diff
@@ -0,0 +1,548 @@
...
+id: getting-started
+title: Quick Introduction
+sidebar_label: Quick Introduction
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Atlas CLI is an open-source tool designed to help software engineers, DBAs and DevOps
+practitioners to manage their database schemas.  Atlas users can use the [Atlas DDL](../concepts/ddl.md)
```
Comment (rotemtam) [inline doc/md/getting-started/getting-started.mdx:11]: i'd link to the SQL guide as its more practical. the DDL doc is a bit low level and not relevant for someonenew

File: doc/md/getting-started/getting-started.mdx [hunk 2]
```diff
@@ -0,0 +1,548 @@
...
+Atlas should show you the following plan:
+
+```text
+-- Planned Changes:
+-- Create "post_categories" table
+CREATE TABLE `example`.`post_categories` (`post_id` int NOT NULL, `category_id` int NOT NULL, CONSTRAINT `post_category_post` FOREIGN KEY (`post_id`) REFERENCES `example`.`blog_posts` (`id`), CONSTRAINT `post_category_category` FOREIGN KEY (`category_id`) REFERENCES `example`.`categories` (`id`))
+✔ Apply
+```
+
+
```
Comment (rotemtam) [inline doc/md/getting-started/getting-started.mdx:546]: newlines

File: doc/website/docusaurus.config.js [hunk 3]
```diff
@@ -122,6 +122,10 @@ module.exports = {
       '@docusaurus/plugin-client-redirects',
       {
         redirects: [
+          {
```
Comment (rotemtam) [inline doc/website/docusaurus.config.js:125]: what about 01 and 02>

File: doc/md/getting-started/getting-started.mdx [hunk 4]
```diff
@@ -0,0 +1,294 @@
...
+<Tabs
+defaultValue="mysql"
+values={[
+{label: 'MySQL', value: 'mysql'},
+{label: 'MariaDB', value: 'maria'},
+{label: 'PostgreSQL', value: 'postgres'},
+{label: 'SQLite', value: 'sqlite'},
+]}>
+<TabItem value="mysql">
```
Comment (a8m) [inline doc/md/getting-started/getting-started.mdx:117]: This "tutorial" uses mysql. No need for the rest of the dialects

**Review by a8m — APPROVED** [review]

**Pull request merged**